frontend/document/canvascontext.lua

```diff
@@ -0,0 +1,61 @@
+local Mupdf = require("ffi/mupdf")
+
+local CanvasContext = {
+    should_restrict_JIT = false,
+    is_color_rendering_enabled = false,
+    is_bgr = false,
+}
+
+--[[
+Initialize CanvasContext with settings from device.
```

Comment:
Some more general comment about the why and what of this module will help future readers.

Reply:
True, some of the stuff from https://github.com/koreader/koreader/pull/4580#issuecomment-462659161 can probably be c/p-ed nearly verbatim.

Reply:
(One does see my comments before I submit the whole review ?! :)

Reply:
There's "add single comment" and "add a review". I don't think it matters too much one way or the other.

Reply:
yeah, if you use add review, it will batch the update. you probably clicked add single comment mistakenly :P